Integrations/DomainTools_Iris/DomainTools_Iris.py

```diff
@@ -0,0 +1,646 @@
+from CommonServerPython import *
+import requests
+
+# disable insecure warnings
+requests.packages.urllib3.disable_warnings()
+
+''' GLOBALS '''
+
+BASE_URL = 'http://api.domaintools.com'
+USERNAME = demisto.params().get('username')
+API_KEY = demisto.params().get('apikey')
+RISK_THRESHOLD = int(demisto.params().get('risk_threshold'))
+YOUNG_DOMAIN_TIMEFRAME = int(demisto.params().get('young_domain_timeframe'))
+
+''' HELPER FUNCTIONS '''
+
+
+def http_request(method, path, other_params=None):
+    """
+    HTTP request helper function
+    Args:
+        method: HTTP Method
+        path: part of the url
+        other_params:
+
+    Returns: request result
+
+    """
+    params = {'app_partner': 'demisto',
+              'app_name': 'Iris Plugin',
+              'app_version': '1',
+              'api_username': USERNAME,
+              'api_key': API_KEY}
+    if other_params:
+        params.update(other_params)
+    url = '{}{}'.format(BASE_URL, path)
+    res = requests.request(
+        method=method,
+        url=url,
+        params=params
+    )
+    res_json = {}
+    try:
+        res_json = res.json()
+    except json.JSONDecodeError as json_error:
+        demisto.error(json_error)
+
+    if not res.ok:
+        error_message = res_json.get('error').get('message')
+        txt = 'error in URL {} status code: {} reason: {}'.format(url, res.status_code, error_message)
+        demisto.error(txt)
+
+    return res_json.get('response')
+
+
+def get_dbot_score(proximity_score, age, threat_profile_score):
+    """
+    Gets the DBot score
+    Args:
+        proximity_score:
+        age:
+        threat_profile_score:
+
+    Returns: DBot Score
+
+    """
+    if proximity_score > RISK_THRESHOLD or threat_profile_score > RISK_THRESHOLD:
+        return 'Bad'
+    elif age < YOUNG_DOMAIN_TIMEFRAME and (proximity_score < RISK_THRESHOLD or threat_profile_score < RISK_THRESHOLD):
+        return 'Suspicious'
+    else:
+        return 'Good'
+
+
+def find_age(create_date):
+    """
+    Finds how many days old a domain is given a start date.
+    Args:
+        create_date: Date in the form of %Y-%m-%d'
+
+    Returns: Number of days
+
+    """
+    time_diff = datetime.now() - datetime.strptime(create_date, '%Y-%m-%d')
+    return time_diff.days
+
+
+def get_threat_component(components, threat_type):
+    """
+    Gets a certain threat component out a list of components
+    Args:
+        components: List of threat components
+        threat_type: Type of threat we are looking for
+
+    Returns: Either the component that we asked for or None
+
+    """
+    for component in components:
+        if component.get('name') == threat_type:
+            return component
+    else:
+        return None
+
+
+def convert_empty_to_null(data_obj):
```

Comment:
this function leads to return `null` to the war room and it doesn't look good, or to the context, which loads it with empty values
in case of null, the relevant cell in the table should be either empty or not appear at all, and should be populated to the context data

Reply:
Changed the function to prune any empty or null attributes.